data-processing/tests/test_parse_equation.py

```diff
@@ -86,6 +86,22 @@ def test_get_symbol_children():
     assert t in t_sub_i.children
     assert i in t_sub_i.children
 
+def test_get_symbol_sub_super_with_numeric_child():
+    with open(
+        get_test_path(os.path.join("mathml", "x_sub_four_sub_three.xml"))
+    ) as mathml_file:
+        mathml = mathml_file.read()
+        symbols = get_symbols(mathml)
+
+    assert len(symbols) == 3
```

Comment:
This is a great test. It makes me realize, do we want `4_3` to be detected as a symbol?

I feel like I we probably don't want `4^3` to be detected as a symbol---that's just a number. `4_3` has no special meaning, and probably won't come up in papers. Though I'm curious what your thoughts are. I'm thinking we may want to add a rule to skip `msub` and `msup` elements that have an `mn` as their base.

Reply:
I think I mostly agree. However, I am interested to know the situations in which 4_3 (or for that matter any calculation involving just numeric symbols) would not have been reduced? My gut says that if there is an expression that has intentionally not been reduced, it holds a special significance in the paper and thus might be something someone wants to actually look for. I'm unsure, what do you think?

Reply:
That's fair! Okay, then let's keep numbers that have a superscript or subscript. We can decide when testing out the tool what feels right.